modules/ROOT/pages/parse-template-reference.adoc

```diff
@@ -51,26 +51,184 @@ Attributes of the `parse-template` element:
 
 
 [[code_example]]
-== Code Example
+== Examples
 
-The following example demonstrates the use of a Parse Template in an application
-that accepts queries by `employeeID` to acquire data about an employee.
+The examples use Parse Template to load data dynamically into HTML.
 
-In this case, Parse Template uses a file external to the flow as a template into which it inserts values for fields (`name`, `department`, `job title`, `start date`, and `employee type`) drawn from the message payload. The flow then returns the template-built payload to the caller.
+=== Use Parse Template to Process a DataBase Query
 
-image::parse-template-flow.png[]
+This example uses Parse Template in a Mule application that loads employee data from a database into an HTML template. The flow returns the template-built output to the caller.
+
+image::parse-template-flow.png[width="55%"]
+
+==== Input to Parse Template
+
+Assume that you start the flow with a URI that contains a query parameter that identifies the ID of an employee, such as `+http://localhost:8081/getEmployee?id=1234+`, and that the Select operation returns the following input to Parse Template:
+
+[source,html,linenums]
+----
+{
+  "first_name": "Melba",
+  "last_name": "Roy Mouton",
+  "department": "Computers",
+  "job_title": "Assistant Chief of Research Programs",
+  "start_date": "01/01/2021",
+  "employee_type": "mathematician"
+}
+----
+
+[[template]]
+==== Template
+
+The template selects fields containing name, department, job title, start date, and employee type values from the input payload.
 
 [source,html,linenums]
 ----
 include::{examplesdir}/responseHtml.template[]
 ----
 
+==== Configuration XML
+
+In the Mule configuration XML for the example, Parse Template loads the template file through the setting, `location="src/main/resources/responseHtml.template"`.
```

Comment:
delete comma after "setting" (there's no pause, the location is just part of the modifying phrase "the template file").